src/components/video-container/Video-container.component.ts

```diff
@@ -258,6 +269,22 @@ export class VideoContainer extends LitElement {
       levelLoadingMaxRetry: 4,
       backBufferLength: navigator.userAgent.match(/Android/i) ? 0 : 30,
       liveDurationInfinity: true,
+      emeEnabled: !!this.drmOptions,
+      drmSystems: this.drmOptions
+        ? {
+            "com.apple.fps": {
+              licenseUrl: this.drmOptions[KeySystems.fps].licenseUrl,
```

Comment:
`DRMOptions` type declares that all keys are optional, should we double check here if all option that we are going to use are actually exists ? Or if we suppose that client should pass all keys to options - maybe we should reflect it in the type ?

Reply:
Basically, all options should be there, but added safe navigation to conform Record behaviour